include/xen.h

```diff
@@ -6,6 +6,13 @@
 #define __XEN_H__
 
 void xen_init(void);
```

Comment:
Are you going to comment this func as well? It looks a bit odd that we comment only one function, but not the other

Reply:
Ok, will do